pkg/generators/openapi3/tests/flow-no-input.hcl

```diff
@@ -0,0 +1,10 @@
+protobuffers = ["./*.proto"]
+
+endpoint "CreateUser" "http" {
+  endpoint = "/user"
+  method = "post"
+}
+
+flow "CreateUser" {
+  output "com.semaphore.User" {}
+}
```

Comment:
new line is missing?

Reply:
well, in the other HCL files there were also no new lines... (I just copied simple.hcl and removed the input block)